documentation/modules/deploying/con-fips-support.adoc

```diff
@@ -1,9 +1,9 @@
 // This assembly is included in the following assemblies:
 //
-// assembly-using-the-cluster-operator.adoc
+// assembly-deploy-intro.adoc
 
 [id='assembly-fips-support-{context}']
-= FIPS support
+= Automatic FIPS support
```

Comment:
Not sure I would call it automatic TBH.

Reply:
How about _Seamless FIPS integration_

Reply:
I would probably leave out the adjectives or how is it called to be honest. The point is that not much of what Strimzi does there is really seemless or automatic. We do not do anything. We really just upgraded to the right OpenJDK version and that is it. Also, I would be a bit reluctant to call it seamless with the non-functioning JMX for example.